src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="post" class="form"
+    onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <lable class="field" for="user_surname">
+          Surname:
+          <input type="text"
+          id="user_surname"
+          name="user_surname"
+          minlength="2"
+          maxlength="50"
+          autocomplete="off"
+          required>
```

Comment:
```suggestion
          <input
            type="text"
            id="user_surname"
            name="user_surname"
            minlength="2"
            maxlength="50"
            autocomplete="off"
            required
          >
```
Fix formatting everywhere in your code. If there are many attributes and you need to put them on a new line each, you should use formatting like this.
You may read more about it here:
https://mate-academy.github.io/style-guides/htmlcss.html